openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -914,6 +914,25 @@ namespace detail {
     bool result = setString(OS_AirLoopHVACFields::DesignSupplyAirFlowRate, "autosize");
     OS_ASSERT(result);
   }
+  
+  /** Baseline Pressure Drop Adjustment **/
+  // Return the Baseline Pdrop
+  double AirLoopHVAC_Impl::baselinePressureDropAdjustment() const {
+    return getDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment,true);
+  }
+  
+  // Set the Baseline Pdrop
+  bool AirLoopHVAC_Impl::setBaselinePressureDropAdjustment(double baselinePressureDropAdjustment) {
+    result = setDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment, baselinePressureDropAdjustment);
+    OS_ASSERT(result);
+  }
+  
+  // Reset Baseline Pdrop
+  void AirLoopHVAC_Impl::resetBaselinePressureDropAdjustment() {
+    bool result = setDouble(OS_AirLoopHVACFields::BaselinePressureDropAdjustment, 0);
```

Comment:
Looks good, often "reset" means resetting the field to an empty string, that would be equivalent in this case since there is a default of 0.  Personally I would reset to an empty string since that is the way the object would be default constructed but I think @kbenne or @mbadams5 might have different views.